katacomb/katacomb/tests/test_continuum_pipeline.py

```diff
@@ -511,7 +511,8 @@ def test_gains_export(self):
 
         # Simulate a '10Jy' source at the phase center
         cat = katpoint.Catalogue()
-        cat.add(katpoint.Target("Alberich, radec, 20.0, -30.0, (856. 1712. 1. 0. 0.)"))
+        cat.add(katpoint.Target(
```

Comment:
Great test - is it worth testing handling of duplicate target names here, or is this tested elsewhere ?

Reply:
Yup. Thats tested on lines: [181-190](https://github.com/ska-sa/katsdpcontim/pull/10/files#diff-5e30dde9bf81af549bed71ca8d5d6889L181-L190) in `test_online_pipeline`.